frontend/tests/controllers/UserRegistrationTest.php

```diff
@@ -186,4 +186,163 @@ public function testUserParentalTokenNotGeneratedDueInvalidParameters() {
             $this->assertSame('parameterInvalid', $e->getMessage());
         }
     }
+
+    /**
+     * User13 can't perform the restricted actions
+     * An exception throws
+     */
+    public function test1User13NotAbleToPerforSomeActions() {
```

Comment:
`test1` doesn't quite tell other folks what this test does. can these test be renamed so that they clearly state what they are testing?

```suggestion
     * Under13 users can't create contests.
     */
    public function testUserUnder13CannotCreateContests() {
```

also, the tests should be in the file most related to the thing that is being tested: this is testing `Contest::apiCreate`, so this should go in `frontend/tests/controllers/ContestCreateTest.php`. Same with all the other tests.